### PR TITLE
Shadcn migration - alert

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Adds the new shadcn Alert component.

### CODE REVIEW

**Comment by pettinarip** [comment]
@nloureiro tagging you since this needs design review.

**Review by Wachira3712 — APPROVED** [review]

**Comment by nloureiro** [comment]
@pettinarip this one is for the alerts on top?
Screen Shot 2024-08-14 12 17 18 AM

like this one?
https://deploy-preview-13617--ethereumorg.netlify.app/en/community/events/

**Comment by pettinarip** [comment]
> @pettinarip this one is for the alerts on top? Screen Shot 2024-08-14 12 17 18 AM
> 
> like this one? https://deploy-preview-13617--ethereumorg.netlify.app/en/community/events/

No, we are not currently using them on prod. You can see them only on Chromatic/Storybook for now. You can see them on the UI tests check or in this [link](https://63b7ea99632763723c7f4d6b-buylkmllza.chromatic.com/?path=/story/molecules-action-feedback-alerts--variants) for the Storybook instance for this build.

**Review by nloureiro — APPROVED** [review]

**Comment by nloureiro** [comment]
All looks good on my end. they look even better

Just one question: why the <pre> on the text for `alert` `success` and `error`
Screen Shot 2024-08-15 09 06 03 AM

**Comment by pettinarip** [comment]
> All looks good on my end. they look even better
Great!

> Just one question: why the

Ah hahaha, I don't know, was me doing silly things. Wanted to highlight the code name for each variant. Will remove the `pre` tag.

File: src/styles/global.css [hunk 1]
```diff
@@ -148,7 +148,9 @@
     /* Complementary Set */
     --attention-outline: var(--attention-light);
 
-    --error-light: var(--error-light);
+    --error: var(--red-500);
+    --error-light: var(--red-100);
+    --error-outline: var(--error);
```
Comment (wackerow) [inline src/styles/global.css:153]: Curious the reason for the `-outline` token variants?

File: src/components/ui/alert.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  React.ButtonHTMLAttributes<HTMLButtonElement>
+>(({ className, ...props }, ref) => (
+  <Button
+    ref={ref}
+    variant="ghost"
+    className={cn("-me-4 rounded-full text-body", className)}
+    {...props}
+  >
+    <MdClose className="h-6 w-6" />
+    <span className="sr-only">Close</span>
```
Comment (wackerow) [inline src/components/ui/alert.tsx:87]: Nice (ref for anyone else: https://tailwindcss.com/docs/screen-readers)

**Review by wackerow — APPROVED** [review]
Very nice 💪 Looks great! Definitely an improvement for this component

File: src/styles/global.css (hunk 1 above)
Comment (pettinarip) [reply src/styles/global.css:153]: Hmm good call, we are not using it, right? I guess the idea was to use this color as the `border` color. I missed that.

**Pull request merged**